### ISSUE TITLE
Same IP Every Time I Connect

### ISSUE THREAD

**Issue by cwz6h** [vpnhood/VpnHood#121]

**Comment by trudyhood** [comment]
Yes, it is behavior by design. When you assign to a server and reconnect, you will be redirected to the same server. You may connect to another server depending on the workload of servers in a Server Farm, but you need to disconnect and wait for more than 15 minutes. VpnHoood Access Server enforces this behavior.
